x-pack/plugins/fleet/common/constants/epm.ts

```diff
@@ -23,16 +23,10 @@ export const STANDALONE_RUN_INSTRUCTIONS_WINDOWS = '.\\elastic-agent.exe install
 
 /*
  Package rules:
-|               | unremovablePackages | defaultPackages | autoUpdatePackages |
-|---------------|:---------------------:|:---------------:|:------------------:|
-| Removable     |         ❌             |        ✔️        |          ✔️         |
-| Auto-installs |         ❌             |        ✔️        |          ❌         |
-| Auto-updates  |         ❌             |        ✔️        |          ✔️         |
-
-`endpoint` is a special package. It needs to autoupdate, it needs to _not_ be
-removable, but it doesn't install by default. Following the table, it needs to
-be in `unremovablePackages` and in `autoUpdatePackages`, but not in
-`defaultPackages`.
+|               | autoUpdatePackages |
+|---------------|:------------------:|
+| Auto-installs |          ❌         |
```

Comment:
nit: let's just remove this line completely since we no longer support this for any packages.

Reply:
done